apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/AppConfiguration.java

```diff
@@ -117,6 +117,24 @@
    */
   public static final String ENV_VAR_KEY_IDEMPOTENCY_REQUIRED = "IDEMPOTENCY_REQUIRED";
 
+  /**
+   * The name of the environment variable that should be used to provide the {@link
+   * LoadAppOptions#isFilteringNonNullAndNon2022Benes()} value, which is a bit complex; please see
+   * its description for details.
+   */
+  public static final String ENV_VAR_KEY_RIF_FILTERING_NON_NULL_AND_NON_2022_BENES =
+      "FILTERING_NON_NULL_AND_NON_2022_BENES";
+
+  /**
+   * The default value to use for the {@link #ENV_VAR_KEY_RIF_FILTERING_NON_NULL_AND_NON_2022_BENES}
+   * configuration environment variable when it is not set.
+   *
+   * <p>Note: This filtering option (and implementation) is an inelegant workaround, which should be
+   * removed as soon as is reasonable. Until then, though, we want it to be turned on in all
+   * environments, so we default it to enabled.
+   */
+  public static final boolean DEFAULT_RIF_FILTERING_NON_NULL_AND_NON_2022_BENES = false;
```

Comment:
This mentions we are defaulting to enabled, but its set to false here.

Reply:
Not good.

Reply:
Addressed in new commit: fixed the comment, left this default the same, and started setting the value in Ansible for our AWS environments.